pkgs/development/python-modules/traits/default.nix

```diff
@@ -17,15 +20,19 @@ buildPythonPackage rec {
     hash = "sha256-qbv9ngwIt94H6G72TmnLlqKcIQWkO/gyzYsWL6HiL0Q=";
   };
 
-  # Circular dependency
-  doCheck = false;
+  patches = [
+    (fetchpatch {
+      url = "https://github.com/enthought/traits/pull/1767/commits/58d4ea897a35e3f285944ed5c3ee6632c32acc94.patch";
```

Comment:
Fetch https://github.com/enthought/traits/commit/a20f2154b2c79eb8550ea9228d1a4415ff51b72a and use includes/excludes instead.